pyaerocom/aeroval/helpers.py

```diff
@@ -130,3 +147,56 @@ def _get_min_max_year_periods(statistics_periods):
         if perstop > stopyr:
             stopyr = perstop
     return startyr, stopyr
+
+
+def _make_dummy_model(obs_list: list, cfg) -> str:
```

Comment:
Are this functions private? If so, why they are been used on a different module?
Maybe the should regular functions on a private module.

``` python
# _dummy_model.py

def add_dummy_model_to_configuration(obs_list: list, cfg: TypeHintPlease) -> str:
    ...

def remove_dummy_model_files(model_id: str) -> None:
    ...

Reply:
Good point, this is just a remnant of an old implementation, where _make_dummy_model was in the experiment_processor. But since it has been moved to the helpers, I'll remove the _